README.md

```diff
@@ -68,11 +69,10 @@ craftguide.add_progressive_filter("Hide secretstuff", function(recipes)
 end)
 ```
 
-#### `craftguide.set_progressive_filter(name, function(recipes, player))`
```

Comment:
A `set_*_filter` function must be kept to override all other functions at once.

Reply:
How about `remove_all_filters`? To keep it doing one thing.

Reply:
No.